core/math/rect2.h

```diff
@@ -405,8 +405,8 @@ struct _NO_DISCARD_ Rect2i {
 		}
 #endif
 		return (p_rect.position.x >= position.x) && (p_rect.position.y >= position.y) &&
-				((p_rect.position.x + p_rect.size.x) < (position.x + size.x)) &&
-				((p_rect.position.y + p_rect.size.y) < (position.y + size.y));
+				((p_rect.position.x + p_rect.size.x) <= (position.x + size.x)) &&
+				((p_rect.position.y + p_rect.size.y) <= (position.y + size.y));
```

Comment:
For the reference, same change was done for `Rect2` in https://github.com/godotengine/godot/pull/32478